server/migrations/20190402105847-create-rating.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Ratings', {
+      user_id: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        primaryKey: true,
+      },
+      article_id: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        primaryKey: true,
+      },
+      rating_value: {
+        type: Sequelize.INTEGER,
+        allowNull: true,
+        defaultValue: null,
+        validate: { min: 0, max: 5 },
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+        defaultValue: Sequelize.fn('now'),
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+        defaultValue: Sequelize.fn('now'),
+      },
+    });
+  },
+  down: queryInterface => queryInterface.dropTable('Ratings'),
+};
```

Comment:
Same as 👇

Reply:
I'm not sure why. I pulled branch develop, checked out my branch and rebased. I didn't have to touch any of these files. Those files are what @McHardex i worked on. I will have to rebase again since his branch has been merged.

Reply:
@rotimi-babalola I have resolved this.